sdk/kiosk/src/query/kiosk.ts

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import {
+  JsonRpcProvider,
+  PaginationArguments,
+  SuiAddress,
+} from '@mysten/sui.js';
+
+import { DynamicFieldPage } from '@mysten/sui.js/src/types/dynamic_fields';
```

Comment:
Export this from the main package if you need it, don't import deeply from the package